inference-engine/src/plaidml_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2018-2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "PlaidMLPlugin")
+
+file(GLOB SOURCES
+        ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/ops/*.cpp
+        ${OS_SPECIFIC_SRC}
+)
+
+file(GLOB HEADERS
+        ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp
+)
+
+addVersionDefines(plaidml_plugin.cpp CI_BUILD_NUMBER PLAIDML_VERSION)
+
+add_library(plaidml SHARED IMPORTED)
+
+set_target_properties(plaidml PROPERTIES IMPORTED_LOCATION ${PLAIDML_SRC_PATH}/bazel-bin/plaidml/libplaidml.so)
```

Comment:
rename PLAIDML_SRC_DIR and make a new property PLAIDML_SO_PATH that defaults to this - I can do this in a follow up too

Reply:
Would be good to make use of: https://github.com/plaidml/plaidml/blob/plaidml-v1/plaidml/plaidml-config.cmake

Or basically tweak both sides so they are compatible with each other.